.gitignore

```
@@ -17,6 +17,7 @@ cmake_install.cmake
 !.vscode/extensions.json
 *.swp
 *~
+cashworkspace.code-workspace
```

I am no expert on vscode (thankfully) but I guess this is more generic:
```suggestion
*.code-workspace
```